db/migrate/20150215074541_create_region_trail_links.rb

```diff
@@ -1,9 +0,0 @@
-class CreateRegionTrailLinks < ActiveRecord::Migration
-  def change
-    create_table :region_trail_links do |t|
-      t.integer :region_id
```

Comment:
Somehow we need to change the scrape/ rake file to add region Id to the
trails??
 On 1 Mar 2015 13:19, "Clement Choi" <notifications@github.com> wrote:

> In db/migrate/20150215074541_create_region_trail_links.rb
> <https://github.com/cchchoi1986/hikehk/pull/65#discussion_r25564186>:
>
> > @@ -1,9 +0,0 @@
> > -class CreateRegionTrailLinks < ActiveRecord::Migration
> > -  def change
> > -    create_table :region_trail_links do |t|
> > -      t.integer :region_id
>
> agree
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/65/files#r25564186>.
>